packages/extensions-sdk/src/cli/commands/helpers/generate-bundle-entrypoint.ts

```diff
@@ -9,28 +9,33 @@ export default function generateBundleEntrypoint(mode: 'app' | 'api', entries: E
 
 	const entriesForTypes = entries.filter((entry) => isIn(entry.type, types));
 
-	const imports = entriesForTypes.map(
-		(entry, i) =>
-			`import e${i} from './${pathToRelativeUrl(
-				path.resolve(
-					isTypeIn(entry, HYBRID_EXTENSION_TYPES)
-						? mode === 'app'
-							? entry.source.app
-							: entry.source.api
-						: entry.source
-				)
-			)}';`
-	);
-
-	const exports = types.map(
-		(type) =>
-			`export const ${pluralize(type)} = [${entriesForTypes
-				.map((entry, i) =>
-					entry.type === type ? (mode === 'app' ? `e${i}` : `{name:'${entry.name}',config:e${i}}`) : null
-				)
-				.filter((e): e is string => e !== null)
-				.join(',')}];`
-	);
+	const imports = entriesForTypes.map((entry, i) => {
+		let entryPath: string;
+
+		if (isTypeIn(entry, HYBRID_EXTENSION_TYPES)) {
+			entryPath = mode === 'app' ? entry.source.app : entry.source.api;
+		} else {
+			entryPath = entry.source;
+		}
+
+		return `import e${i} from './${pathToRelativeUrl(path.resolve(entryPath))}';`;
```

Comment:
Let's write out `i` here as well, like further down in `types.map`
```suggestion
	const imports = entriesForTypes.map((entry, index) => {
		let entryPath: string;

		if (isTypeIn(entry, HYBRID_EXTENSION_TYPES)) {
			entryPath = mode === 'app' ? entry.source.app : entry.source.api;
		} else {
			entryPath = entry.source;
		}

		return `import e${index} from './${pathToRelativeUrl(path.resolve(entryPath))}';`;
```